File: samples/Java/Json Store/Jsql/Find.java
String brand = "Bally";

String sql = "SELECT P.$.name name"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.brand)=LOWER(:brand)";

final StringBuilder pinballs = new StringBuilder();
int count = session.openJsqlCursor().database("playground")
			.query(sql).param("brand",brand)
			.find(new JsqlHandler() {
				public boolean handle(JsqlEntry entry) throws JsonException {
					pinballs.append("\n"+entry.getColumn("name"));
					return true;
				}
			});

_formatText("Pinballs Count:{0}{1}",count,pinballs);

File: samples/JavaScript/Json Store/Jsql/Find.js
session.setAsync(false);

var brand = "Bally";

var sql = "SELECT P.$.name name"
	+" FROM pinball P"
	+" WHERE LOWER(P.$.brand)=LOWER(:brand)";

var pinballs = "";
var count = session.openJsqlCursor().database("playground")
			.query(sql).param("brand",brand)
			.find(function(entry) {
				pinballs += "\n"+entry.getColumn("name");
				return true;
			});

darwino.Utils.setText("content","Pinballs Count:{0}{1}",count,pinballs);